src/looprestoration_tmpl_8.rs

```diff
@@ -288,6 +288,30 @@ extern "C" {
     );
 }
 
+#[cfg(all(feature = "asm", target_arch = "arm"))]
+extern "C" {
+    fn dav1d_wiener_filter_h_8bpc_neon(
+        dst: *mut int16_t,
+        left: *const [pixel; 4],
+        src: *const pixel,
+        stride: ptrdiff_t,
+        fh: *const [int16_t; 8],
```

Comment:
What you had previously, `*const int16_t`, was correct, as `const int16_t fh[8]` in an argument decays to `const int16_t *fh`, and that's how `c2rust transpile` translates it.